wagtail/images/templates/wagtailimages/images/results_image.html

```diff
@@ -15,4 +15,8 @@
 
 {% load wagtailimages_tags %}
 
-<div class="image">{% image image max-165x165 class="show-transparency" %}</div>
+{% with image.width|stringformat:"s" as image_width and image.height|stringformat:"s" as image_height %}
+    {% with image_description=image.title|add:" ("|add:image_width|add:" pixels by "|add:image_height|add:" pixels)" %}
```

Comment:
I think we might want to drop the `image.title` completely, because of #5350. This PR could solve both issues at the same time.

For `pixels by` – do you know how we could make this text translateable?

Reply:
I think it might be best in that case if we moved the image dimensions from the alt text to the title below, and leave the alt text blank. It depends if you think that sighted users get enough info about the image dimensions from looking at the thumbnail, and therefore don't need that information as much as users who can't see the thumbnail. To me though it makes sense to keep the title and image dimensions together, rather than separating them out.

Reply:
I will investigate about translating the text. Do you think using `x` and `px` rather than `by` and `pixels` would be more language independent? I used the latter ones because it sounded better when read aloud.

Reply:
I was wondering the same thing yesterday, I think even "x" is too english-specific. Screen readers also tend to mispronounce very short words / abbreviations like this.

Reply:
Presumably a Unicode × (multiplication sign) character is a better choice than an "x" :-)

Reply:
OK so perhaps the answer is to go with `pixels` translated and the unicode sign, thanks @gasman

Reply:
@thibaudcolas please can you review my latest commit addressing the points above?